apps/site/lib/site_web/views/helpers/time_helpers.ex

```diff
@@ -1,9 +1,18 @@
 defmodule SiteWeb.TimeHelpers do
+  @moduledoc """
+  Shortcuts to manipulate DateTimes and Dates
+  """
```

Comment:
In the case that the moduledoc fits on one line, we don't have to use the multi-line docstring format:

```elixir
@moduledoc "Shortcuts to manipulate DateTimes and Dates"
```